test/plugins/validation/swagger2/operations-ibm.js

```diff
@@ -503,4 +503,43 @@ describe('validation plugin - semantic - operations-ibm - swagger2', function()
     expect(res.errors.length).toEqual(0);
     expect(res.warnings.length).toEqual(0);
   });
+
+  it('should not complain about a get operation id contains a hyphen', function() {
+    const config = {
+      operations: {
+        operation_id_naming_convention: 'warning'
+      }
+    };
+
+    const spec = {
+      paths: {
+        '/CoolPath': {
+          get: {
+            consumes: ['application/json'],
+            produces: ['application/json'],
+            summary: 'this is a summary',
+            operationId: 'list-cool_path',
```

Comment:
Do you operation IDs use mixed case? It seems strange to have underscores and hyphens. I know it's just a test but I would prefer to have a better example of a kebab-case ID.

```suggestion
            operationId: 'list-cool-path',
```

Reply:
This was simply an example to assure that - and _ are acceptable. As for the need, you're right about the need, so I changed it.